inference/core/workflows/core_steps/sinks/roboflow/custom_metadata/v1.py

```diff
@@ -151,7 +151,7 @@ def run(
             api_key=self._api_key,
             inference_ids=inference_ids,
             field_name=field_name,
-            field_value=field_value[0],
+            field_value=field_value,
```

Comment:
seems liek good change, but I dont understand why it would have worked before version change.

@PawelPeczek-Roboflow as part of your versions PR loosk liek you changed ti from 
```
async def run(
        self,
        fire_and_forget: bool,
        field_name: str,
        field_value: Batch[str],
        predictions: Batch[sv.Detections],
    )
 ```
 
 ```
 def run(
        self,
        fire_and_forget: bool,
        field_name: str,
        field_value: str,
        predictions: sv.Detections,
    )
 ```
 
 Maybe it was batch order before and not anymore?  just don't understand how always taking [0] would have worked properly before then.